fairscale/optim/utils.py

```diff
@@ -4,12 +4,18 @@
 # LICENSE file in the root directory of this source tree.
 
 import io
-from typing import Any, Dict
+from typing import TYPE_CHECKING, Any, Dict, List
 
 import torch
 from torch._six import container_abcs
 import torch.distributed as dist
 
+if TYPE_CHECKING:  # pragma: no cover
```

Comment:
Why is this conditional?

Reply:
copy paste from oss.py, my guess was that it was to catch strict or loose type hinting